300_demons/100_post_demons.md

```diff
@@ -4,6 +4,11 @@
 
 ## `POST`{.verb} `/api/v2/demons/`
 
+<div class='info-yellow'>
+<b>Rate Limits:</b><br>
+This endpoint is ratelimited at 1 request per minute
```

Comment:
whoa, is it? why'd I do that :joy:

Reply:
ah, did i miss something? seems like it's mentioned in these two places:
[pointercrate-demonlist-api/src/endpoints/demon.rs#L91](https://github.com/stadust/pointercrate/blob/a3683bdab3c028398e5b702079df1570fd943782/pointercrate-demonlist-api/src/endpoints/demon.rs#L91)
[pointercrate-demonlist-api/src/ratelimits.rs#L13](https://github.com/stadust/pointercrate/blob/a3683bdab3c028398e5b702079df1570fd943782/pointercrate-demonlist-api/src/ratelimits.rs#L13)

Reply:
ahhh, I remember now. When I look it over yesterday I thought this was `GET /demons`, not `POST`, my bad